### PR TITLE
Formalize contingency and summary of the organization itself.

### PR DESCRIPTION
**Pull request by calmofthestorm**
With discussion of Caster joining dictation-toolbox, it's probably a good idea for us to formalize our agreement on exactly what dictation-toolbox is for. I've written this up as my best recollection of what we agreed to over email forever ago, but obviously I think the current admins should reach a consensus on this before we consider it binding.

I'm pretty flexible on the four months (I'd probably be fine with anything >=2 and <=12.)

I considered adding language about making changes to this document, and maybe we should, but my main concern is to formalize our ownership of the projects and all aspects of community maintenance belonging to the owners, rather than to the admins of dictation-toolbox, and I really want to keep the bureaucracy and process light weight.

I care a great deal about maintaining my control over my projects but not much about my control in the organization itself (if there were ever to be an unresolvable disagreement, for example, I'd likely be content to move my projects out of the org, and just leave a link in place and an agreement not to fork into where the repo used to be to avoid confusion).

But honestly if voice coding ever gets popular to the point where we reach disagreements that would be an awesome problem to have in a sense, since I feel like the much bigger threat is the transient nature and generally small size of the community.

So basically I'm not convinced we need to address this now, provided we're clear about repository ownership and "loose association", and set out clear terms for the contingency option.

nirvdrum@ I'll treat your approval as required on this, and will also reach out to the two private-membership admins, though I'm not anticipating any objections.

### CODE REVIEW

**Comment by calmofthestorm** [comment]
@LexiconCode @Danesprite possibly of interest to you.

**Comment by calmofthestorm** [comment]
Ironically I accidentally merged the OWNERS addition to dragonfly-scripts rather than getting @nirvdrum 's approval as I intended. Mentioning this here to ensure this is okay!

**Comment by drmfinlay** [comment]
Thanks for thinking of me. The summary looks good to me and I think that contingency plan is a good idea.

I'm open to migrating my dragonfly fork to dictation-toolbox under those terms if you like, after discussing it over there first of course.

**Comment by calmofthestorm** [comment]
I'd love to welcome Dragonfly as well, since it's arguably one of the most common dependencies of all our various projects. I'll open an issue there to discuss it.

**Comment by LexiconCode** [comment]
It's good to see things formalized as I appreciate clear expectations. Thank you @calmofthestorm for making this merger possible.

**Comment by calmofthestorm** [comment]
I'm planning to merge this without review on Jan 2. I've done my best to contact the other admins first to give them time. I will address their concerns after the fact if any arise.

**Comment by nirvdrum** [comment]
Sorry for just getting to this. I've been traveling a bit for the holiday season and my laptop is currently bricked. I'll try to look at this in the next day or two.

**Comment by nirvdrum** [comment]
Well, that was a quick read. Sorry for taking so long to get to it. And no worries about the OWNERS file.

I think I have more to say about the purpose of the organization, but I'm happy to add that in in another PR. Hopefully it's not too controversial, since it's my own rationale for why I wanted to create this org :-)

I think we can introduce a little bit of process to better define the bounds expected by admins. In particular, I think we should require each repo to have a GitHub team and the members of that team are understood to be the only ones to take administrative action of that repo. This could extend to having the ability to merge PRs, close issues, etc.

I'm not married to that idea though. The current approach we've adopted has been fine and I'm not expecting an organizational scaling crisis any time soon. I'm also happy to let this PR merge and we can discuss the team idea separately.

**Comment by calmofthestorm** [comment]
I'm fine with formalizing more autonomy than guaranteed by this PR. I mostly don't want less:-) Anyway, since you're ok with it, let's merge this and move dragonfly2 and caster in, and we can flesh out further organization together as desired.
